schemas/2016-10-01/Microsoft.StorSimple.1200.json

```diff
@@ -0,0 +1,1545 @@
+{
```

Comment:
Looks like we already have a definition for this resource type here: https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2016-10-01/Microsoft.StorSimple.json.

Please could you remove this to avoid duplication?